config/default/plugin-site.yaml

```diff
@@ -26,12 +20,11 @@ ingress:
 
 restApiUrl: https://plugins.jenkins.io/api
 
-replicaCount: 2
+backend:
+  replicaCount: 2
 
-resources:
-  limits:
-    cpu: 2000m
-    memory: 2048Mi
-  requests:
-    cpu: 1000m
-    memory: 1024Mi
+htmlVolume:
+  azureFile:
+    secretName: plugin-site
+    shareName: prodpluginsite
```

Comment:
shareName 'pluginsite' as defined [here](https://github.com/jenkins-infra/azure/blob/936993e6306247920552c84d659106a93ca0d01d/plans/pluginsite.tf#L28)

Reply:
```suggestion
    shareName: pluginsite
```

Reply:
sooooo, i don't think thats true actually.

https://github.com/jenkins-infra/azure/blob/936993e6306247920552c84d659106a93ca0d01d/plans/pluginsite.tf#L6

21:34 <rtyler> our Azure resources are prefixed with prod fwiw

https://github.com/jenkins-infra/plugin-site/blob/master/Jenkinsfile#L74-L84

Reply:
the resource group is prefixed but the share is hardcoded

Reply:
okay, the naming scheme for azure seems different with each tool, so i'll defer

Reply:
ShareName is one level below than the azure file storage.
shareName name needs to be unique per azure file storage, where azure file storage need to be unique.